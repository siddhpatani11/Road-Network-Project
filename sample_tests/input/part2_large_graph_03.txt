121 447
0 16 14 G
0 90 30 G
0 106 32 R
1 86 30 G
1 92 40 R
1 93 37 B
1 93 2 R
1 108 49 G
1 108 17 B
2 58 8 B
2 61 35 R
3 26 6 G
3 26 13 G
3 64 46 G
3 109 16 G
3 112 41 G
5 27 17 R
5 27 21 G
5 39 35 G
5 75 3 B
6 15 27 G
6 16 39 G
6 69 40 B
6 99 41 B
7 16 9 R
7 17 5 G
7 44 7 R
7 119 37 B
8 100 24 R
9 8 30 G
10 11 2 R
10 70 25 G
10 74 20 B
10 74 7 R
11 106 44 G
11 106 51 G
12 19 13 G
12 19 19 B
12 25 8 R
12 47 10 R
12 49 37 B
12 97 12 R
13 16 49 G
13 91 30 G
13 91 17 G
14 19 14 B
14 29 28 R
14 29 50 R
14 54 50 B
14 75 33 R
14 102 24 G
15 9 50 G
15 23 36 G
15 56 30 R
15 75 32 B
15 106 6 R
15 106 30 B
16 2 19 B
16 9 15 G
16 68 44 G
17 30 29 G
17 87 1 R
17 92 14 R
17 119 18 R
18 67 13 B
18 68 32 R
18 82 51 R
18 111 3 R
19 18 16 G
19 18 14 R
19 36 6 B
19 38 19 G
19 71 19 G
20 6 2 G
20 45 39 B
20 46 22 G
20 73 43 B
21 4 12 B
21 4 11 B
21 34 17 G
21 66 20 B
22 0 36 G
22 0 21 G
22 10 14 G
22 34 11 G
22 45 4 R
22 77 31 B
23 7 45 G
23 120 33 B
24 36 23 G
24 36 14 G
24 77 13 G
25 14 19 B
25 62 37 G
25 65 5 G
25 74 34 G
25 75 12 B
25 83 15 R
26 27 25 B
26 42 29 G
26 75 11 B
27 32 5 R
27 40 17 B
27 57 33 G
27 65 20 B
27 65 43 G
28 5 44 B
28 26 36 R
28 73 31 R
28 75 19 G
28 99 15 R
28 106 50 B
28 106 4 R
29 22 18 R
29 35 37 R
29 40 16 G
29 40 49 R
30 7 12 B
30 23 44 B
30 114 48 G
30 114 9 B
31 29 12 G
31 41 45 B
31 59 15 R
31 112 10 G
32 21 19 B
32 40 28 R
32 40 21 B
32 42 18 B
32 42 6 G
33 64 40 R
34 53 42 R
34 61 33 R
34 61 47 G
34 78 8 G
36 23 26 B
36 49 26 B
36 68 35 B
36 89 12 R
36 92 44 R
37 60 47 B
37 67 23 R
38 24 23 G
38 24 2 G
38 44 34 R
38 44 9 R
38 66 40 G
38 74 41 G
38 83 5 R
38 83 28 R
38 84 17 R
38 103 35 B
38 110 7 G
38 110 22 G
39 57 42 R
39 66 23 G
39 82 2 R
39 99 43 B
39 108 19 G
39 108 3 G
40 51 22 B
40 72 38 R
40 109 28 G
41 22 47 B
41 78 42 G
41 78 43 R
42 40 33 B
42 54 20 R
42 59 31 B
42 96 48 B
43 24 41 B
43 77 51 R
43 113 16 R
44 5 37 G
44 69 31 R
44 69 40 R
45 60 39 B
45 116 7 B
45 116 26 R
47 35 28 B
47 62 51 G
47 118 7 B
48 63 13 B
48 85 44 B
48 85 41 G
49 7 3 B
49 18 24 B
49 117 43 B
49 117 40 B
50 54 7 G
50 64 10 G
50 64 29 G
50 79 27 R
50 79 28 R
51 0 10 G
51 16 9 R
51 16 1 B
51 50 3 G
51 50 26 G
52 4 43 G
52 74 32 G
52 108 29 R
53 52 33 R
53 57 13 G
53 57 16 B
53 61 45 B
54 2 4 B
54 7 38 R
54 18 44 B
54 62 23 B
55 74 47 R
55 95 25 B
56 1 42 B
56 5 23 G
57 89 42 B
58 40 43 R
58 40 25 R
58 45 19 G
58 56 38 R
58 89 47 G
58 105 11 R
58 120 48 G
58 120 27 B
59 61 28 B
60 65 49 G
60 93 33 R
61 29 19 R
61 97 5 B
61 110 16 G
61 110 23 R
62 94 30 G
62 95 47 G
62 95 22 B
63 15 16 R
63 43 13 R
63 94 35 B
63 94 24 B
63 110 37 R
63 110 33 R
64 4 27 R
64 72 27 R
65 9 41 G
65 23 15 R
65 26 23 B
65 53 50 R
65 56 21 R
65 56 16 G
65 76 50 B
65 95 33 G
65 96 24 G
65 101 48 G
65 101 3 G
66 86 24 B
66 86 30 R
66 112 8 G
66 118 38 B
67 65 8 G
67 65 10 R
68 8 49 B
68 30 7 R
68 92 18 R
68 109 41 R
68 109 48 B
68 113 3 B
69 18 2 R
69 36 51 G
69 36 43 B
70 27 29 B
70 57 2 R
70 86 12 G
70 86 44 B
70 91 29 B
71 80 15 G
71 90 32 B
71 106 39 R
71 106 34 B
71 108 46 B
72 19 7 B
72 20 18 G
72 46 41 B
72 94 33 B
72 94 47 G
73 95 18 B
73 114 46 G
74 25 47 G
74 33 17 R
74 70 35 B
74 94 18 G
75 107 10 R
75 112 37 G
75 116 28 B
76 28 37 R
76 77 25 B
77 15 11 R
77 62 2 R
77 62 29 B
77 64 50 B
77 69 42 B
77 109 36 G
78 13 3 R
78 51 41 G
78 51 49 G
79 19 18 B
79 46 41 R
79 46 23 B
79 94 44 R
80 32 51 G
80 65 24 R
80 97 29 G
81 118 47 R
82 68 4 R
82 73 22 B
82 99 44 R
83 27 4 R
83 27 21 R
83 36 35 R
83 74 30 R
83 103 29 B
83 107 46 R
84 21 14 R
84 52 40 B
84 94 29 G
86 14 4 G
86 65 10 R
87 22 24 B
87 61 40 G
87 65 51 G
87 112 31 B
88 56 13 B
88 101 10 B
88 101 24 B
88 102 45 R
89 5 50 B
89 35 14 R
89 56 8 G
89 56 8 G
89 69 49 B
89 69 49 R
89 109 39 G
89 109 32 G
90 30 35 B
90 83 26 B
90 83 30 G
90 85 30 R
91 18 7 B
91 20 38 B
91 64 8 B
91 69 10 G
92 36 21 R
92 36 14 B
93 115 15 G
94 18 2 B
94 19 21 G
94 23 42 B
94 40 16 B
94 50 26 B
95 8 7 G
95 32 44 B
95 34 18 R
95 35 35 R
95 44 36 R
95 44 43 R
96 39 17 B
96 39 36 R
96 45 48 B
96 45 9 R
96 52 20 B
96 60 48 R
96 72 12 B
96 72 29 R
96 90 2 G
97 53 6 R
97 53 18 R
97 66 48 R
97 66 34 R
98 104 29 G
98 104 35 B
98 106 8 R
98 107 7 G
98 112 24 G
98 112 30 R
100 43 25 B
100 59 39 B
100 70 48 R
100 92 20 R
100 92 4 B
102 27 39 B
102 28 11 B
102 57 3 B
102 73 39 G
102 109 12 G
102 109 16 G
102 113 16 R
103 82 12 R
103 82 40 G
104 8 35 G
104 40 17 B
104 40 26 G
104 93 49 B
105 64 3 G
105 72 29 G
105 72 34 R
105 99 42 R
106 21 25 B
106 78 19 R
106 82 18 R
107 6 10 G
107 49 38 R
107 74 32 R
107 108 39 B
108 37 38 R
108 84 1 R
108 99 24 B
108 120 24 B
109 3 4 R
109 15 10 R
109 39 21 G
109 56 22 B
109 64 44 R
109 64 26 B
110 7 29 B
110 18 47 R
110 28 31 G
110 38 37 G
110 48 10 B
110 48 12 R
110 83 5 R
111 23 33 G
111 62 33 B
112 8 6 R
112 40 29 R
112 50 9 B
114 86 6 G
115 38 21 R
116 71 18 G
116 97 23 R
116 97 35 G
117 1 29 R
118 45 31 R
118 114 34 G
119 4 9 G
119 4 22 B
119 54 6 R
120 24 12 G
120 48 9 G
120 85 21 B
120 111 7 G
