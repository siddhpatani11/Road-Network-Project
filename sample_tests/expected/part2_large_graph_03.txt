81 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - 0 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 - -1 R 81 - -1 - -1 - -1 R 47 - -1 - -1 
